Doc/library/gzip.rst

```diff
@@ -211,3 +211,38 @@ Example of how to GZIP compress a binary string::
       The basic data compression module needed to support the :program:`gzip` file
       format.
 
+Command Line Interface
+----------------------
+
+The :mod:`gzip` module provides a simple command line interface to compress or
+uncompress files.
```

Comment:
Same here to use decompress instead of uncompress.

Reply:
fixed